lib/Driver/Compilation.cpp

```diff
@@ -1645,6 +1680,13 @@ namespace driver {
           .findExternallyDependentUntracedJobs(externalDependency);
     }
 
+    std::vector<const Job *>
+    markIncrementalExternalInDepGraph(StringRef externalDependency,
```

Comment:
the name says mark, but it doesn't look like it marks.
If there is marking, is that a part of the interface, or just the implementation?

Reply:
It was named by analogy with the non-incremental case. Does that no longer do marking?